cmd2.py

```diff
@@ -1188,6 +1411,7 @@ def do_shortcuts(self, args):
     def do_eof(self, arg):
         """Called when <Ctrl>-D is pressed."""
         # End of script should not exit app, but <Ctrl>-D should.
+        print('')  # Required for clearing line when exiting submenu
```

Comment:
If you could find a clever way so that this conditionally executes based on some attribute which is only present within a submenu, that would be preferable just out of paranoia for it potentially having a negative effect on someones existing scripts and/or transcript tests. 

 If not, it seems like a pretty minor change which I don't think would be relevant for most end users.